### PR TITLE
Specify optional setting of `submission_date` by importer (#40)

### LINKED ISSUE #40: Specify optional setting of `submission_date` by importer
https://humancellatlas.slack.com/archives/C01360XN04S/p1628006310005200

### PR DESCRIPTION
**Pull request by hannes-ucsc**

### CODE REVIEW

File: docs/dcp2_system_design.rst [hunk 1]
```diff
@@ -913,12 +913,60 @@ Importer idempotence
...
+Some sources may find it difficult to provide a meaningful value for both
+``submission_date`` and ``update_date`` when they populate a staging area
+with updated entities. The ``submission_date`` in the new version of an
+updated entity needs to be set to the same value as in the original version,
+and that typically requires that the original is available to the source.
+Stateful sources, like Ingest, that keep track of entities in a database will
+have access to the original. Stateless sources, like Analysis, would need to
+retrieve it from TDR. To make their life easier, the importer applies a small
+transformation before importing an updated entity. In the pseudo-code below,
+``orginal`` denotes the version of the entity already present in TDR while
```
Comment (aherbst-broad) [inline docs/dcp2_system_design.rst]:
```suggestion
``original`` denotes the version of the entity already present in TDR while
```

**Review by aherbst-broad — APPROVED** [review]

**Review by jessicaway — COMMENTED** [review]
This looks good to me. I would like to resolve the question of how the adapter workflow will set submission date before officially approving (see [slack thread here](https://broadinstitute.slack.com/archives/C01360XN04S/p1628006310005200))

**Review by ESapenaVentura — APPROVED** [review]
LGTM, I think this is how the exporter currently behaves (AFAIK, @amnonkhen may be able to confirm)

**Review by jessicaway — APPROVED** [review]

**Review by NoopDog — APPROVED** [review]

**Comment by hannes-ucsc** [comment]
Two reminders were sent out:

1st: https://humancellatlas.slack.com/archives/C01360XN04S/p1631581931067400
2st: https://humancellatlas.slack.com/archives/C01360XN04S/p1632238761095800

File: docs/dcp2_system_design.rst (hunk 1 above)
Comment (hannes-ucsc) [reply docs/dcp2_system_design.rst]: Fixed

**Pull request merged**